core/systems/plugin/plugin.gd

```diff
@@ -12,6 +12,7 @@ const OGUIButton := preload("res://core/ui/components/button.tscn")
 const ButtonStateChanger := preload("res://core/systems/state/state_changer.tscn")
 const qam_state_machine := preload("res://assets/state/state_machines/qam_state_machine.tres")
 
+@export var focus_node : Node
```

Comment:
Instead of this, let's add `focus_node` as an argument to the `add_to_qam` method. E.g.
```
func add_to_qam(qam_item: Control, icon: Texture2D, focus_node: Control = null) -> void:
    ...
    qam.add_child_menu(qam_item, icon, focus_node)
```